src/main/java/ru/sbtqa/tag/pagefactory/drivers/TagWebDriver.java

```diff
@@ -259,16 +255,11 @@ private static String parseDriverVersionFromMapping(String browserVersion, Strin
     }
 
     private static JsonObject getResourceJsonFileAsJsonObject(String filePath) {
-        ClassLoader classLoader = TagWebDriver.class.getClassLoader();
-        try {
-            Path file = Paths.get(classLoader.getResource(filePath).toURI());
-            JsonParser parser = new JsonParser();
-            JsonReader reader = new JsonReader(new BufferedReader(new FileReader(file.toFile())));
-            return parser.parse(reader).getAsJsonObject();
-        } catch (URISyntaxException | IOException e) {
-            LOG.error("Error during parsing mapping file", e);
-        }
-        return null;
+        ClassLoader classLoader = ClassLoader.getSystemClassLoader();
+        InputStreamReader isr = new InputStreamReader(classLoader.getResourceAsStream(filePath));
```

Comment:
Does it throw any exceptions?

Reply:
InputStreamReader() throws unchecked in case of wrong encoding but I don't think this would happen

Reply:
The cp1251 encoding is usual error in windows dev environment, it will conflicts in utf8 environment, so I think we should handle these possible situations.